services/auth/src/api.js

```diff
@@ -923,6 +930,8 @@ builder.declare({
   method: 'get',
   route: '/scopes/current',
   name: 'currentScopes',
+  // Not sure if this needs to be scope free or not
```

Comment:
Let's answer this question.. (I'm guessing from the green checkmarks that it's OK?)